helm/README.md

```diff
@@ -2,20 +2,31 @@
 
 1. Create the cluster with [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
 
-   > Make sure ports 30001 and 30002 are free on your machine
+   > Make sure ports 30001, 30002 and 30003 are free on your machine
```

Comment:
30002 = grafana and 30003 = (optionally) the detached integrations server, right?